Sources/Conduit/Networking/Serialization/MultipartFormRequestSerializer.swift

```diff
@@ -26,7 +26,19 @@ public final class MultipartFormRequestSerializer: HTTPRequestSerializer {
     static private let CRLF = "\r\n"
 
     private var formData: [FormPart] = []
-    private let contentBoundary = MultipartFormRequestSerializer.randomContentBoundary()
+
+    lazy var contentBoundary: String = {
+        let letters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+        let lettersLength = UInt32(letters.count)
+
+        let randomCharacters = (0..<12).map { _ -> String in
+            let offset = Int(arc4random_uniform(lettersLength))
```

Comment:
Conduit defines its Swift version to be 5.0, so we can use [SE-0202 "Random Unification"](https://github.com/apple/swift-evolution/blob/master/proposals/0202-random-unification.md).
Unfortunately, it returns optionals. So it would be something like `return String((0..<12).map { _ in letters.randomElement() ?? "a" })`